apps/snitch_core/test/data/model/package_item_test.exs

```diff
@@ -31,102 +29,113 @@ defmodule Snitch.Data.Model.PackageItemTest do
          shipping_category_count: 1,
          shipping_method_count: 1,
          state_zone_count: 1
-    test "fails with invalid params", context do
-      %{line_items: [line_item]} = context
-
-      assert {:error, changeset} =
-               PackageItem.create(%{@params | line_item_id: -1, product_id: -1, package_id: -1})
-
-      refute changeset.valid?
-      assert %{package_id: ["does not exist"]} == errors_on(changeset)
-
-      assert {:error, changeset} =
-               PackageItem.create(%{
-                 @params
-                 | line_item_id: line_item.id,
-                   product_id: -1,
-                   package_id: -1
-               })
-
-      refute changeset.valid?
-      assert %{package_id: ["does not exist"]} == errors_on(changeset)
-
-      assert {:error, changeset} =
+    test "successful with valid attributes", context do
+      %{package_item: package_item} = new_package(context)
+      assert {:ok, _} =
```

Comment:
The approach here is not correct.

You only have to create an empty package, then use the line_item you have set up to create a package item. You are already creating a package_item in `new_package` and using the attributes of that to create another package_item.